politeiawww/api/pi/v1/v1.go

```diff
@@ -243,3 +243,17 @@ type SetBillingStatusReply struct {
 	Receipt   string `json:"receipt"`
 	Timestamp int64  `json:"timestamp"` // Unix timestamp
 }
+
+const (
```

Comment:
The `PolicyReply` should have a comment explaining the naming inconsistency between the pi plugin and the politeiawww pi API for the `Name` vs `Title`.